js/recycle.js

```diff
@@ -0,0 +1,3 @@
+window.onscroll = function() {
```

Comment:
You repeat this for each page. I would recommend simply having a `backgroundscroll.js` file or something like that which you include on every page that needs it.